float32 radius
---
bool result
